### ISSUE TITLE
Support org-habits 

### ISSUE THREAD

**Issue by v-Nyo** [orgzly-revived/orgzly-android-revived#307]
https://orgmode.org/manual/Tracking-your-habits.html
https://orgmode.org/worg/org-tutorials/tracking-habits.html

Would be really amazing

**Comment by amberin** [comment]
Is there any particular org-habit feature that you're missing? The app does support all the basic repeaters, and you can configure it to log completions to a log book.

**Comment by amberin** [comment]
No, you are right, it doesn't support the / syntax, only basic repeaters such as ".+2d". But I was wondering what you miss the miss the most, because I use Orgzly to log habits and it suffices for my particular needs.

**Comment by amberin** [comment]
It's not stupid at all. It's hard to compete with Emacs' customizability, but in theory anything is possible. This software is built by its users.

**Comment by amberin** [comment]
@v-Nyo Yes, please do. Be sure to describe how it would be different from today's search queries, since they already allow creating multiple different agenda views (using the "ad" keyword).

**Comment by jtpavlock** [comment]
I think it would be useful to hide habits on future days in the agenda, mirroring Emacs' behavior. My week agenda view ends up getting cluttered from all the daily habits I have set up.

**Comment by zetashift** [comment]
Are there any tips for a search query where daily habits are hidden?

**Comment by amberin** [comment]
> I think it would be useful to hide habits on future days in the agenda, mirroring Emacs' behavior. My week agenda view ends up getting cluttered from all the daily habits I have set up.

Agreed. Especially for short-term agenda queries, it makes most sense to show only the next upcoming occurrence. And this feels like something you'd want to specify per query, not globally for all queries. So it might actually warrant extending the search query syntax. I can imagine three different choices for notes with a repeater:

1. Show all occurrences
2. Show only the first occurrence
3. Hide all occurrences

Can we think of more?

> Are there any tips for a search query where daily habits are hidden?

I don't believe there is any way to hide them today.
